pages/tx/[hash].tsx

```diff
@@ -272,6 +286,12 @@ const Tx = () => {
     utf8Input ? { type: 'utf8', text: utf8Input } : null,
   ].filter(v => v)
 
+  const getGasPriceDisplayValue = (gasPrice: string): string => {
+    return gasPrice === '0'
```

Comment:
The condition `gasPrice === '0'` could be used in 
1. show `price` or `gasless`
2. show `fee` or `gasless`
3. show `user operations` or not

so I think it could be assigned to a variant